kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -2,22 +2,47 @@
 
   <div class="user-edit-modal">
     <modal btntext="Edit">
-      <div class="title" slot="header">User edit</div>
+
+      <h1 slot="header" class="header">Account Info</h1>
```

Comment:
Wasn't the title "Edit User"?
If this modal opens with the "pencil" button, I believe the modal title should still include the word "edit", maybe "View and Edit Account Info"?